auto-value-gson/build.gradle

```diff
@@ -3,8 +3,8 @@ import org.gradle.internal.jvm.Jvm
 apply plugin: 'java-library'
 apply plugin: 'maven-publish'
 
-sourceCompatibility = JavaVersion.VERSION_1_7
-targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
this is the only change that needs to happen. the rest are not necessary.

Reply:
Oops, missed that part. Yes this part can be reverted. Rest should stay java 7 though for the reasons listed above

Reply:
Note I can't reopen the PR because github says your branch was "force pushed or recreated"